pkg/asset/machines/master.go

```diff
@@ -624,9 +624,9 @@ func (m *Master) Load(f asset.FileFetcher) (found bool, err error) {
 	file, err = f.FetchByName(filepath.Join(directory, controlPlaneMachineSetFileName))
 	if err != nil {
 		if os.IsNotExist(err) {
-			return false, nil
+			return true, nil
```

Comment:
From https://github.com/openshift/installer/blob/master/pkg/asset/machines/master.go#L599-L600, returning true here when the file is clearly not found is probably incorrect.

The function calling Load() should be modified to not throw out an error (identified in the bug) when `found` is false but `err` is nil. That approach seems consistent with the description provided in this PR that, CPMS files is not mandatory.

Reply:
UPI needs CPMS files to be deleted to work. So it's more like mandatory but not mandatory.

Reply:
Is there a way we can identify if the manifest is required or not and skip this block instead of relying on a combination of return values?

Reply:
All these run agnostic of cloud platform or installation type so adding these checks will be real hard. Ignoring CPMS if it doesn't exist for now seems like an easier choice.

Reply:
So at least a comment explaining why we're returning `true` even though the file doesn't exist or there was an error loading it? Maybe also a debug log message for cases when we expect those files to be loaded?

Reply:
Sounds good. Added comment and log.